Sql_Assignment

```diff
@@ -0,0 +1,310 @@
+| 1153     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_3 | Demo Product Unit Three | 1.000000 |    14.99000 |
+| 1153     | 03                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_4 | Product Unit FourDemo   | 1.000000 |    12.99000 |
+| 1255     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_4 | Demo Product Unit Four  | 3.000000 |    12.99000 |
+| 1061     | 02                | PRODUCT_ORDER_ITEM | 02                | DEMO_UNIT_5 | Demo Product Unit Five  | 1.000000 |    55.09000 |
+| 1064     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_5 | Demo Product Unit Five  | 3.000000 |    12.07000 |
+| 1255     | 03                | PRODUCT_ORDER_ITEM | 02                | DEMO_UNIT_5 | Demo Product Unit Five  | 3.000000 |    10.99000 |
++----------+-------------------+--------------------+-------------------+-------------+-------------------------+----------+-------------+
+
+10). List the Person details whose employment status is either EMPS_SELF or EMPS_FULLTIME without using the OR operator.
+
+Select * from Person where EMPLOYMENT_STATUS_ENUM_ID IN ("EMPS_FULLTIME", "EMPS_SELF");
++-----------+------------+-------------+-----------+--------+------------+---------------------------+-------------------+
+| PARTY_ID  | FIRST_NAME | MIDDLE_NAME | LAST_NAME | GENDER | BIRTH_DATE | EMPLOYMENT_STATUS_ENUM_ID | OCCUPATION        |
++-----------+------------+-------------+-----------+--------+------------+---------------------------+-------------------+
+| CustDemo1 | Joe        | Q           | Public    | M      | 1999-06-15 | EMPS_FULLTIME             | Software Engineer |
+| CustDemo2 | Jack       | John        | Smith     | M      | 1999-09-14 | EMPS_FULLTIME             | Doctor            |
+| CustDemo4 | John       | J           | Norvig    | F      | 1998-06-11 | EMPS_FULLTIME             | Software Engineer |
+| CustDemo5 | Charlie    | Jack        | William   | M      | 1992-04-22 | EMPS_SELF                 | Architect         |
+| CustDemo6 | James      | W           | Evans     | M      | 1993-05-20 | EMPS_FULLTIME             | Civil Engineer    |
++-----------+------------+-------------+-----------+--------+------------+---------------------------+-------------------+
+
+11). List all columns of Facility table.
+
+Select * from Facility;
++--------------------+---------------------+
+| FACILITY_ID        | FACILITY_NAME       |
++--------------------+---------------------+
+| RegionalWarehouse1 | Regional Warehouse1 |
+| RegionalWarehouse2 | Regional Warehouse2 |
+| WebStoreWarehouse  | Web Store Warehouse |
++--------------------+---------------------+
+
+12).Output the sum of the grandTotal from OrderHeader table for each order date.
+
+select sum(Grand_Total) AS Grand_Total from Order_Header where grand_Total IN (select Grand_Total from Order_Header group by Order_Date);
```

Comment:
Not the right query for this question.

Reply:
SELECT order_date, SUM(grand_total) FROM order_header GROUP BY order_date;

Reply:
| order_date              | SUM(grand_total) |
+-------------------------+------------------+
| 2019-04-24 00:00:00.000 |          87.0700 |
| 2020-03-17 00:00:00.000 |          75.9300 |
| 2020-04-17 00:00:00.000 |         224.1900 |
| 2020-04-19 00:00:00.000 |          31.6400 |
| 2022-01-09 00:00:00.000 |          48.8700 |
+-------------------------+------------------+